<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-U7LEM7P" failures="2" tests="2" name="RetrymultipleFailedTestCases" time="0.053" errors="0" timestamp="2020-08-07T23:00:39 IST">
  <testcase classname="myTest.MyTestcases" name="test1" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at myTest.MyTestcases.test1(MyTestcases.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="myTest.MyTestcases" name="test2" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at myTest.MyTestcases.test2(MyTestcases.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- RetrymultipleFailedTestCases -->
